### PR TITLE
Fix client server connection

### PR DESCRIPTION
**Pull request by ramy-att**

### CODE REVIEW

**Comment by ramy-att** [comment]
> #### ⚠️ GitGuardian has uncovered 1 secret following the scan of your pull request.
> Please consider investigating the findings and remediating the incidents. Failure to do so may lead to compromising the associated services or software components.
> 
> 🔎 Detected hardcoded secret in your pull request
> 
> GitGuardian id	Secret	Commit	Filename	
> [5496914](https://dashboard.gitguardian.com/incidents/5496914?occurrence=85701133)	MongoDB Credentials	[b14280a](https://github.com/ramy-att/trivial-soen341project2023/commit/b14280ad8360c1d358d7ef711e64b637c4ddd4b4)	server/index.js	[View secret](https://github.com/ramy-att/trivial-soen341project2023/commit/b14280ad8360c1d358d7ef711e64b637c4ddd4b4#diff-bcb729747c92e72c12e4590f736a334041a572c429756da8af60b6ab89c6be79L18)
> 🛠 Guidelines to remediate hardcoded secrets
> 
> 1. Understand the implications of revoking this secret by investigating where it is used in your code.
> 2. Replace and store your secret safely. [Learn here](https://blog.gitguardian.com/secrets-api-management?utm_source=product&utm_medium=GitHub_checks&utm_campaign=check_run_comment) the best practices.
> 3. Revoke and [rotate this secret](https://docs.gitguardian.com/secrets-detection/detectors/specifics/mongo_uri#revoke-the-secret?utm_source=product&utm_medium=GitHub_checks&utm_campaign=check_run_comment).
> 4. If possible, [rewrite git history](https://blog.gitguardian.com/rewriting-git-history-cheatsheet?utm_source=product&utm_medium=GitHub_checks&utm_campaign=check_run_comment). Rewriting git history is not a trivial act. You might completely break other contributing developers' workflow and you risk accidentally deleting legitimate data.
> 
> To avoid such incidents in the future consider
> 
> * following these [best practices](https://blog.gitguardian.com/secrets-api-management/?utm_source=product&utm_medium=GitHub_checks&utm_campaign=check_run_comment) for managing and storing secrets including API keys and other credentials
> * install [secret detection on pre-commit](https://https://docs.gitguardian.com/ggshield-docs/integrations/git-hooks/pre-commit?utm_source=product&utm_medium=GitHub_checks&utm_campaign=check_run_comment) to catch secret before it leaves your machine and ease remediation.
> 
> 🦉 [GitGuardian](https://dashboard.gitguardian.com/auth/login/?utm_medium=checkruns&utm_source=github&utm_campaign=cr1) detects secrets in your source code to help developers and security teams secure the modern development process. You are seeing this because you or someone else with access to this repository has authorized GitGuardian to scan your pull request.Our GitHub checks need improvements? [Share your feedbacks](https://050127dayho.typeform.com/to/KmeAPTMk)!

It's okay for now - np

File: server/index.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
 const mongoose = require("mongoose");
+const cors = require("cors");
 const express = require("express");
+
 const studentRouter = require("./routes/student-routes");
 
-const PORT = process.env.PORT || 3001;
```
Comment (TarekElalfi) [inline server/index.js:5]: Would that affect anything in the backend?

**Review by TarekElalfi — APPROVED** [review]

File: server/index.js (hunk 1 above)
Comment (ramy-att) [reply server/index.js:5]: We can return it and remove the PORT declared on line 7

**Comment by TarekElalfi** [comment]
Great job Ramy, Thanks for the amazing work.

**Pull request merged**